source/system/external/stdatomic.ooc

```diff
@@ -0,0 +1,27 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+include ./atomic
+
+atomic_int: extern cover
```

Comment:
Huh, first time I've seem `extern cover` use, I knew it was valid syntax but wasn't too sure it would behave as expected.  

Any particular reason this is not `atomic_int: cover from atomic_int`?  
By the way, you could define the extern C functions directly inside your types if you did something like:  

```ooc
// atomic_bool never defined in ooc source before
AtomicBool: cover from atomic_bool {
    init: extern(atomic_init_bool) func@ (Bool)
    set: extern(atomic_store_bool) func@ (Bool)
    // Etc.
    // Operators could also be nice
    operator@ = (b: Bool) -> This {
        set(b)
        this
    }
}
```

Of course, this API is also useful to have defined, so I guess it boils down to stylistic choice.